neo/Network/P2P/Payloads/DisconnectPayload.cs

```diff
@@ -0,0 +1,53 @@
+using Neo.IO;
+using System.IO;
+
+namespace Neo.Network.P2P.Payloads
+{
+    public enum DisconnectReason : byte
+    {
+        MaxConnectionReached = 0x01,
+        MaxConnectionPerAddressReached = 0x02,
+        DuplicateConnection = 0x03,
+        MagicNumberIncompatible = 0x04,
+        ConnectionTimeout = 0x05,
+        UntrustedIpAddresses = 0x06,
```

Comment:
I think we should disconnect from the node directly without sending the reason for `UntrustedIpAddresses`.

Reply:
Good suggestion 👍 
Otherwise you leak information an attacker can use to improve his odds for success.

Reply:
fixed